src/frontend/Komponenter/Behandling/VedtakOgBeregning/Overgangsstønad/InnvilgeVedtak/InntektsperiodeValg.tsx

```diff
@@ -91,6 +91,17 @@ const ReadMoreMedMarginBottom = styled(ReadMore)`
     margin-bottom: 2rem;
 `;
 
+const SamordningsfradragTypeContainer = styled.div`
+    margin: 1.5rem 0;
```

Comment:
I stedet for å lage nye komponenter som setter sin egen margin, kan du gjøre style `HorizontalScroll` i denne komponenten til å bli en fleksboks med gap: 1rem; for eksempel

Reply:
Støttes veldig, men er så mye ulik spacing i denne at fordelen med gap blir borte:((